korio/src/commonMain/kotlin/com/soywiz/korio/time/TraceTime.kt

```diff
@@ -13,7 +13,7 @@ inline fun <T : Any> traceTime(name: String, block: () -> T): T {
         result = block()
     }
     if (traceTimes) {
-        Console.info("$name loaded in $time")
+        Logger("TraceTime").info { "$name loaded in $time" }
```

Comment:
Maybe @PublishedAPI internal val traceTimeLogger outside the function?

Reply:
done